src/wallet/rpcwallet.cpp

```diff
@@ -2901,6 +2905,16 @@ static UniValue listunspent(const JSONRPCRequest& request)
         entry.pushKV("scriptPubKey", HexStr(scriptPubKey.begin(), scriptPubKey.end()));
         entry.pushKV("amount", ValueFromAmount(out.tx->tx->vout[out.i].nValue));
         entry.pushKV("confirmations", out.nDepth);
+        if (!out.nDepth) {
```

Comment:
`AvailableCoins` filters negative values but that takes a little code snooping to assure the reader. Can we make it easier on the reader and make this `out.nDepth == 0`?

Reply:
That doesn't change it at all... Negative values would be handled the same.